### PR TITLE
Added the efs operations

### PR DESCRIPTION
**Pull request by athiruma**
## Type of change
Note: Fill **x** in []
- [ ] bug
- [x] enhancement
- [ ] documentation
- [ ] dependencies

### CODE REVIEW

File: cloud_governance/common/clouds/aws/efs/efs_operations.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
+from cloud_governance.common.clouds.aws.utils.common_methods import get_boto3_client
+
+
+class EFSOperations:
+
```
Comment (ebattat) [inline cloud_governance/common/clouds/aws/efs/efs_operations.py:5]: Can u pls add the class documentation
What the meaning of EFS = Amazon Elastic File System (EFS) and the purpose of this class?

**Review by ebattat — CHANGES_REQUESTED** [review]

File: cloud_governance/common/clouds/aws/efs/efs_operations.py (hunk 1 above)
Comment (athiruma) [reply cloud_governance/common/clouds/aws/efs/efs_operations.py:5]: ok, this class helps to delete the EFS resources in zombie_cluster.

Comment (athiruma) [reply cloud_governance/common/clouds/aws/efs/efs_operations.py:8]: done.

**Review by ebattat — APPROVED** [review]
/LGTM

**Pull request merged**